packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -74,9 +74,10 @@ async function createDataObject<
 		runtimeClass,
 	);
 
-	// Create a new runtime for our data store
+	// Create a new runtime for our data store, as if via new FluidDataStoreRuntime,
+	// but using the runtimeClass that's been augmented with mixins
 	// The runtime is what Fluid uses to create DDS' and route to your data store
-	const runtime: FluidDataStoreRuntime = new runtimeClass(
+	const runtime: FluidDataStoreRuntime = new runtimeClass( // calls new FluidDataStoreRuntime(...)
```

Comment:
Adding these comments because I keep searching code for "new FluidDataStoreRuntime" and can't find it and then have to remember about the mixins and go find this code.  Now searching for constructor call will find this code :)

Reply:
I wish VS Code + TS server could identify this code when you find usages of the constructor, but it doesn't.  Finding usage of the class itself works, but apparently I personally don't jump to try that.